gnd/src/test/java/com/google/android/gnd/persistence/local/room/converter/ResponseJsonConverterTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.local.room.converter;
+
+
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.model.form.Field.Type;
+import com.google.android.gnd.model.observation.DateResponse;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.model.observation.TimeResponse;
+import com.google.android.gnd.persistence.local.LocalDatabaseModule;
+import com.google.android.gnd.rx.SchedulersModule;
+import com.google.common.truth.Truth;
+import dagger.hilt.android.testing.HiltAndroidTest;
+import dagger.hilt.android.testing.HiltTestApplication;
+import dagger.hilt.android.testing.UninstallModules;
+import java.util.Date;
+import java8.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@HiltAndroidTest
+@UninstallModules({SchedulersModule.class, LocalDatabaseModule.class})
+@Config(application = HiltTestApplication.class)
+@RunWith(RobolectricTestRunner.class)
+public class ResponseJsonConverterTest {
+
+  public String dateString = "2021-09-24T22:10+0530";
+  public Date date = new Date(1632501600000L);
```

Comment:
Please make these `private static final`, capitalize name, and add comment explaining what date/time`1632501600000L` refers to.

Reply:
updated https://github.com/google/ground-android/pull/846/commits/a950d1b98f2dfde1ae77921c5ffd9d9d2a3b732b